src/index.ts

```diff
@@ -191,6 +187,43 @@ client.on('voiceStateUpdate', async (oldState, newState) => {
   }
 });
 
+// Event triggered when a user's presence (e.g. status, activity, etc.) is changed.
+client.on("presenceUpdate", async (_, newPresence) => {
+  // Determine if this presence update is because a user is now playing a game
+  if (newPresence.activities[0]?.type === ActivityType.Playing) {
```

Comment:
Thinking right now that there should be an extra check added here to verify that the user associated with this presence update is in the same `VoiceChannel` as the bot to make sure other users in the server aren't affecting the starting/stopping of the polling.